tests/test_babs_check_setup.py

```diff
@@ -53,6 +59,8 @@ def test_babs_check_setup(
     tmp_path_factory: fixture from pytest
```

Comment:
Dorota, could you please add some docs here re: the arguments you added for this function?